contents/handbook/people/spending-money.md

```diff
@@ -185,14 +185,16 @@ We have a company WeWork All Access account that allows you to book desks in any
 
 ## Travel
 
-If you need to travel on PostHog’s behalf (e.g. in person onboarding, meeting customers, working on a specific project), again please spend money in the best interests of the company. 
+If you need to travel on PostHog’s behalf for in-person onboarding, meeting customers, and offsites, again please spend money in the best interests of the company. 
 
 We do not pre-approve your travel expenses, and trust team members to make the right decisions here. Some guidance:
 - We all travel economy by default- we cannot afford for folks to fly premium or business class. Feel free to upgrade using your personal money/airmiles if you’d like to. If you are particularly tall and struggle to fit into an economy seat, PostHog will cover the cost of an emergency exit seat. 
 - Be sensible when considering the value of your time, and avoid over-optimizing for cost. It is probably not worth saving $100 to take a non-direct flight that takes you 5hrs extra. Similarly, it doesn’t make sense to save $50 by taking a flight that leaves at 6am, only to have to expense a taxi to get to the airport.
 - Related to the above, consider signing up for programs like Global Entry if you are regularly travelling to countries that offer it- these can pay for themselves in one trip with the amount of time saved, particularly when travelling to the US. PostHog will cover costs here.
 - When travelling internationally, PostHog does not cover roaming charges for your phone. You can expense a reasonable eSIM, which usually is no more than $20. 
 
+> If you find yourself needing to do extra travel outside of the regular things listed above, e.g. you've been asked to take a last minute trip to work on an emergency project, we may have some flexibility here, especially if you are travelling at very short notice or long haul. Talk to Fraser in the first instance if you think this may apply to you. This is intended for genuine one-offs, not where you've decided you'd like to come along to an extra offsite!
```

Comment:
Curious, has there been an (anonymized) example of this happening recently? I had a very similar situation with a last-minute project, though that didn't end up materializing

Reply:
Yep your situation would probably have been such a case! (pending discretion, cost of the flight etc etc)

I think the difference is usually 'James/Tim have asked me to fly out last minute to work on a thing'.

Reply:
i'd have said no because your trip was super short. this is more for "yo you are in california, could you appear in london in a day or two please?"

Reply:
Yes, so my feedback here is that I do not understand what "flexibility" could possibly apply in such an urgent&important case (beside the inherent fact of traveling on short notice overall).

Reply:
Ah right - I meant 'flexibility' as in 'booking you a nicer seat' but it sounds like that wasn't clear, is that what you meant by your question?

Reply:
Yup